apps/factory/src/pages/dashboard/prompts/[id]/index.tsx

```diff
@@ -160,12 +161,29 @@ const PackageShow: NextPageWithLayout = () => {
             ) : (
               <span></span>
             )}
-            <CreateTemplate
-              pp={pp as pp}
-              onCreate={ptCreateMutation.mutate}
-              status={status}
-              customError={customError}
-            ></CreateTemplate>
+            <Grid component="span">
+              <Grid container>
+                <Grid item xs={6} md={6} lg={6}>
+                  <CreateTemplate
+                    pp={pp as pp}
+                    onCreate={ptCreateMutation.mutate}
+                    status={status}
+                    customError={customError}
+                    ptId={""}
```

Comment:
use false instead of empty string

Reply:
- using a boolean value will cause problems in fetching template data from the server, the getTemplate function will run as soon as the component gets rendered so we cannot put a condition on it to check string or boolean

Reply:
-writing false will add up extra checks which we can avoid by empty string